src/languages/constants.ts

```diff
@@ -24,4 +25,13 @@ export const supportedLanguageIds = [
   "xml",
 ] as const;
 
+/**
+ * Other language IDs that we might reference (e.g., for token customization)
+ * but don't have full tree-sitter support for yet.
+ */
+export const otherLanguageIds = ["shellscript"] as const;
```

Comment:
I wonder, is this array actually used? Can it just live in the land of types instead?

Reply:
Hm, possibly not. I was just mirroring the existing code. `supportedLanguageIds` is used inside of `isLanguageSupported()`. But that function itself is unused...